replibyte/src/bridge/mod.rs

```diff
@@ -4,11 +4,11 @@ use std::io::Error;
 
 pub mod s3;
 
-struct IndexFile {
+pub struct IndexFile {
     backups: Vec<Backup>,
 }
 
-struct Backup {
+pub struct Backup {
```

Comment:
Rust will not allow publically used private structs in the future. Reference: https://github.com/rust-lang/rust/issues/34537